lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -377,14 +377,14 @@ Ufe::UndoableCommand::Ptr UsdContextOps::doOpCmd(const ItemPath& itemPath)
             return nullptr;
 
         return std::make_shared<AddReferenceUndoableCommand>(
-            fPrim, path);
+            fItem->prim(), path);
     }
     else if (itemPath[0] == ClearAllReferencesUndoableCommand::commandName) {
         MString confirmation = MGlobal::executeCommandStringResult(clearAllReferencesConfirmScript);
         if (ClearAllReferencesUndoableCommand::cancelRemoval == confirmation)
             return nullptr;
 
-        return std::make_shared<ClearAllReferencesUndoableCommand>(fPrim);
+        return std::make_shared<ClearAllReferencesUndoableCommand>(fItem->prim());
```

Comment:
Would have been sweet to create an inline accessor for this, to save some typing, and also possibly to do some null checking:
inline UsdPrim prim() const { TF_AXIOM(fItem != nullptr); return fItem->prim(); }